README.md

```diff
@@ -412,3 +425,27 @@ Note platforms will be tested regularly in the Arm CI system. Please discuss wit
 # Known-issues
 
 Please review existing issues on [GitHub](https://github.com/ARMmbed/mbed-os-example-pelion/issues) and report any problem you may see.
+
+# Migration Guide for Targets supporting FOTA
+
+1. Compilation for the above FOTA supported targets is similar to as UC
+   bootloader. Steps would be to:
+    a. Install the mbed cli as for other targets, please check sections 
+       ## Requirements
+       ## Deploying
+       ## Preparing for build
+2. Once the above binaries are ready following needs to be done to update the
+   binaries to the TARGET's
+    a. The binary need to be truncated to the FLASH size as the KV store needs
+       to be reset to update the KEYS on the board.
+       For e.g. for K66F target following needs to be done as the FLASH size is
+       2M:
+       #truncate <PATH>/mbed-os-example-pelion.bin --size 2M
+    b. Once the BIN is truncated to the desired size, it needs to be flashed to 
+       board.
+    c. Once the BIN is flashed succefully the bootloader is upgraded to FOTA
+       supported bootloader. Now FOTA can be used to upgrade the application
+
+Note: On "How to upgrade the application using the FOTA" please refer
+    https://developer.pelion.com/docs/pelion-client-lite/v1.2/developer-guide/index.html
```

Comment:
```suggestion
1. Compilation for the above FOTA supported targets is similar to as UC
   bootloader. Steps would be to:
    a. Install the mbed cli as for other targets, please check sections 
       ## Requirements
       ## Deploying
       ## Preparing for build
2. Once the above binaries are ready following needs to be done to update the
   binaries to the TARGET's
    a. The binary need to be truncated to the FLASH size as the KV store needs
       to be reset to update the KEYS on the board.
       For e.g. for K66F target following needs to be done as the FLASH size is
       2M:
       #truncate <PATH>/mbed-os-example-pelion.bin --size 2M
    b. Once the BIN is truncated to the desired size, it needs to be flashed to 
       board.
    c. Once the BIN is flashed succefully the bootloader is upgraded to FOTA
       supported bootloader. Now FOTA can be used to upgrade the application

Note: On "How to upgrade the application using the FOTA" please refer
    https://developer.pelion.com/docs/pelion-client-lite/v1.2/developer-guide/index.html
```
1. Official documentation is still work in progress.
2. Current new bootloaders are compiled from https://github.com/PelionIoT/mbed-bootloader/tree/nanoclient feature branch which provide both the application and bootloader kvstore and update configurations.
3. The provided prebuild-bl folder contain several examples for new configuration system and will be expanded in future.